common/lib/xmodule/xmodule/video_module/video_handlers.py

```diff
@@ -166,6 +167,10 @@ def get_static_transcript(self, request, transcripts):
         if not self.transcript_language == 'en':
             return response
 
+        # If this video lives in library, the code below is not relevant and will error.
```

Comment:
Question for reviewers: Should this method (get_static_transcript) ever succeed for a content library? Would a content library ever have "static_asset_path" or "data_dir" (see lines 184-188)?